pkg/raft/transport/http_transport.go

```diff
@@ -0,0 +1,207 @@
+package transport
```

Comment:
can you add some comments to this file - why we need to implement our own transport layer, a short explanation of the RPC type would be nice.

Reply:
Theres a readme that goes into detail about this stuff at the root of this package